manual/src/cmds/runtime.etex

```diff
@@ -136,7 +137,13 @@ The following environment variables are also consulted:
   section~\ref{Hashtbl}).
 \fi
         This option takes no argument.
+  \item[H] Allocates heap chunks by mmapping huge pages. Huge pages are locked into
+    memory, and are not swapped.
   \item[h] The initial size of the major heap (in words).
+  \item[w] Sets size of window used by major GC for smoothing out variations in
+    its workload. This is an integer between 1 and 50. (Default: 1)
+  \item[W] Activates runtime warnings (such as Channel opened on file dies without
+    being closed, unflushed data, etc.)
```

Comment:
```suggestion
    being closed, unflushed data, etc.)
    \item[t] Sets the trace level for the debug runtime (ignored otherwise).
```
Line 181 (`% FIXME missing: H, t, w, W see MPR#7870`) below can then be deleted.

Reply:
Hi David! Thank you for the reviews. 

I have a question about the options of OCAMLRUNPARAM and ocamlrun. 

Why is the sequence of options disordered?  

In the function `caml_parse_ocamlrunparam`, the options are ordered. 
![image](https://user-images.githubusercontent.com/2202271/135106967-597d4347-91d0-49f8-b4c0-58e793a9ae4f.png)

But in the manpage and manual files, the sequence is disordered.  
![image](https://user-images.githubusercontent.com/2202271/135107395-0eba2e27-a445-457f-9aae-c187ce8e2f69.png)